README.md

```diff
@@ -121,7 +121,7 @@ Front-end code (both development and production) is compiled using [Node](http:/
 
 ### Requirements
 
-You need Node (minimum version of 0.10.0, maximum version 0.12.x), which will also get you [NPM](npmjs.org), Node's package management tool. 
+You need 6.10.3 <= node < 7.0.0 which is what we currently use in production.
```

Comment:
Can we clarify why we wouldn't be able to upgrade node beyond 7.0.0 (hmm, what about security fixes?)

Reply:
So my original idea was to enforce the same version we use in production, but that might be a bit too strict. Do you think we should just specify a minimum version?

Reply:
I see what you mean - the ideal DRY thing would be for us to all be developing using docker - the version of node is specified in the Dockerfile after all.

Perhaps it would be clearer to just reference the base image, maybe say something like "You need Node (try to install the version we use in production -
 see the [base docker image](https://github.com/alphagov/digitalmarketplace-docker-base/blob/master/base.docker)."

Reply:
It would be neat if we could get the version from that file, but I couldn't think of a simple way of doing that. I agree your content is better, I'll update it.

We should have a reminder somewhere to update node before April next year: https://github.com/nodejs/Release#release-schedule

Reply:
Hijacked your existing ticket and scheduled it for "later" https://trello.com/c/YDiLYdWQ/181-update-node-to-version-8 :-)